docs/concepts/articulations.rst

```diff
@@ -25,10 +26,43 @@ forward kinematics (:func:`newton.eval_fk`) converts generalized coordinates to
 In Newton, we support both parameterizations and it is up to the solver which one to use to read and write the configuration.
 For example, :class:`~newton.solvers.SolverMuJoCo` and :class:`~newton.solvers.SolverFeatherstone` use generalized coordinates, while :class:`~newton.solvers.SolverXPBD` and :class:`~newton.solvers.SolverSemiImplicit` use maximal coordinates.
 
-When declaring an articulation using the :class:`~newton.ModelBuilder`, the rigid body poses (maximal coordinates :attr:`newton.State.body_q`) are initialized by the ``xform`` argument:
+Let's consider an example where we create an articulation with a single revolute joint and initialize
```

Comment:
Does it make sense to specify this example as how to initialize articulation state using reduced coordinates, vs. the second one that uses maximal coordinates?

Reply:
I've added a few words before this sentence to motivate this example.